metricbeat/docs/modules/mongodb.asciidoc

```diff
@@ -42,6 +42,27 @@ Or more complex like:
   hosts: ["mongodb://myuser:mypass@localhost:40001", "otherhost:40001"]
 ----------------------------------------------------------------------
 
+some more supported URLS are:
+
+[source,yaml]
+----------------------------------------------------------------------
+- module: mongodb
+  hosts: ["mongodb://localhost:27017,localhost:27022,localhost:27023"]
```

Comment:
The only difference when we pass hosts in these 2 ways is:
1. hosts: ["mongodb://myuser:mypass@localhost:40001", "otherhost:40001"] --> In this case we have 2 different service addresses i.e mongodb://myuser:mypass@localhost:40001 and mongodb://myuser:mypass@otherhost:40001 and in ES we will get events from these addresses separately.
2. on the other hand, in this case: hosts: ["mongodb://localhost:27017,localhost:27022,localhost:27023"], it will be a single service address and we will get single event from this address.

Please refer the below screenshots:
![image](https://user-images.githubusercontent.com/102972658/229450037-5ac60886-2054-4b62-9a71-a7d118fd6d89.png)
![image](https://user-images.githubusercontent.com/102972658/229450136-7d7f1ce5-4c67-4433-a8b2-b32525eb8b45.png)